### ISSUE TITLE
weird chrome behavior

### ISSUE THREAD

**Issue by jamuspsi** [seiyria/gameicons-font#1]
In chrome, windows 7, this font seems to add a TON of spacing above the icon, like 300-400px.  It doesn't happen if I switch the font to something like arial, just as a test.  Even the example page looks super wonky with it, with the elements stretching invisibly all the way up to the filter/color inputs.

However, the OTHER version of this, the fork for the idle game, doesn't have this behavior, it seems.  I don't have the first clue about how to fix this (or even open the files), and I'm hoping it's a matter of you just re-saving what you have.  If you need any extra information about it, screenshots, dev tools stuff, please let me know.  Thanks for the cool project- I just wish I could use it!

**Comment by seiyria** [comment]
Yep. So the thing is, I think some of the icons are messed up. Both of my
forks work because I use a small subset of icons, but for some reason it
does this on windows only. It's very frustrating but I basically gave up. I
recommend doing what I do in my forks and choosing a subset of icons and
building your font that way.

I spent about a week trying to figure out why and I came up empty-handed,
so it's gotta stay that way unfortunately. If you figure anything out
though don't hesitate to let me know.
